packages/workflow/src/NodeHelpers.ts

```diff
@@ -1097,14 +1219,14 @@ export function getParameterIssues(nodeProperties: INodeProperties, nodeValues:
 				continue;
 			}
 
-			if (nodeProperties.typeOptions !== undefined && nodeProperties.typeOptions.multipleValues !== undefined) {
+			if (nodeProperties.typeOptions?.multipleValues !== undefined) {
```

Comment:
is not this the same?
```suggestion
			if (nodeProperties.typeOptions?.multipleValues?) {
```

Reply:
From Ben: This is the optional access operator and falsy is not the same as `undefined`.